.github/workflows/build-and-push-container-image.yml

```diff
@@ -17,36 +17,37 @@ jobs:
       - name: Is Versioning Commit
         run: |
             isVersioningCommit=${{ startsWith(github.event.commits[0].message, 'Version NFS Server Chart') }};
-            echo "IS_VERSIONING_COMMIT=$isVersioningCommit" >> $GITHUB_OUTPUT;
-            echo "IS_VERSIONING_COMMIT=$isVersioningCommit";
+            declare -p isVersioningCommit;
+            echo "value=$isVersioningCommit" >> $GITHUB_OUTPUT;
+            echo "value=$isVersioningCommit";
         id: isVersioningCommit
 
       - name: Get Version
         run: |
             version=$(jq -r .version package.json);
-            echo "VERSION=$version" >> $GITHUB_OUTPUT;
-            echo "VERSION=$version";
+            echo "value=$version" >> $GITHUB_OUTPUT;
+            echo "value=$version";
         id: version
 
-      - if: steps.isVersioningCommit.outputs.IS_VERSIONING_COMMIT
+      - if: steps.isVersioningCommit.outputs.value == 'true'
```

Comment:
This is the main fix here - the if statement wasn't evaluating correctly before.